api/main.go

```diff
@@ -35,6 +35,16 @@ import (
 	"github.com/apisix/manager-api/log"
 )
 
+var Version string
+
+func printInfo() {
+	fmt.Fprint(os.Stdout, "The manager-api is running successfully!\n\n")
+	fmt.Fprintf(os.Stdout, "%-8s: %s\n", "Version", Version)
+	fmt.Fprintf(os.Stdout, "%-8s: %s:%d\n", "Listen", conf.ServerHost, conf.ServerPort)
+	fmt.Fprintf(os.Stdout, "%-8s: %s\n", "Loglevel", conf.ErrorLogLevel)
```

Comment:
Log info is described in config file, I think here show listen info is enough because this is the most important running info to user, how do you think?

Reply:
The listen info is also described in the config file. But we show it.
So I think it's more convenient for users to get the log file path location if the file path is described in `relative path` format.

Reply:
Output listen info have two meanings:
- Tell user, manager api is running normal
- Show the serve info

If we output the log info, why we not output other config? We should give a standard to tell other developer what should be print with begin info instead of component's log,I  think `relative path` is not reasonable.